swagger/v2/swagger.json

```diff
@@ -1853,6 +1853,13 @@
             "null"
           ]
         },
+        "appointment_type": {
```

Comment:
This should not be present in V2

Reply:
Updated V2 model to exclude V2 swagger json.